.yaydoc.yml

```diff
@@ -0,0 +1,35 @@
+metadata:
+  author: FOSSASIA
+  projectname: "Susi Server"
+  version: development
+build:
+  theme:
+    name: sphinx_fossasia_theme   
+  source: docs
+  logo: images/susi-logo-original.svg
+  subproject:
+    - url: https://github.com/fossasia/susi_android
+      source: docs
```

Comment:
I didn't make any changes it was already there. I don't know somehow `.yaydoc.yml` got deleted in development branch